robocop/checkers/documentation.py

```diff
@@ -24,6 +24,12 @@
         """,
     ),
     "0202": Rule(
+        RuleParam(
+            name="ignore_templated",
+            default=True,
+            converter=bool,
```

Comment:
It's a common pitfall: bool(str). bool("False") is actually true since it evaluates it as non-empty truthy string. You can use this method instead: https://docs.python.org/3/distutils/apiref.html#distutils.util.strtobool

Reply:
^Above will return int, so you need to convert it to bool.

Optionally write your own converter:
def str2bool(v):
  return v.lower() in ("yes", "true", "t", "1")
  
(from SO ;) https://stackoverflow.com/questions/715417/converting-from-a-string-to-boolean-in-python)

Reply:
Fixed it. Can you take a look?